cmd/jlv/main.go

```diff
@@ -14,6 +14,12 @@ import (
 const configFileName = ".jlv.jsonc"
 
 func main() {
+	//os.Args = append(os.Args, "assets/seconds.log")
+	f, err := tea.LogToFile("debug.log", "debug")
+	if err != nil {
+		fmt.Println("An error occurred!", err.Error())
+	}
+	defer f.Close()
```

Comment:
Will be removing all of the changes here, just leaving this as a WIP. Getting VSCode/delve to work with a bubbletea app was giving me problems so I decided to just move to logging temporarily.